### ISSUE TITLE
Understanding and deactivating lrUpdateRate during word representation training

### ISSUE THREAD

**Issue by orestxherija** [facebookresearch/fastText#1217]
I would like to learn word vectors in an unsupervised fashion using a large corpus. I am trying to understand a bit how the `lrUpdateRate` parameter works. It appears to play the role of a learning rate scheduler but most likely not in the traditional sense, where the scheduler essentially is a function from epochs or batch indices to a real number (the new learning rate). Here the default value for `lrUpdateRate` is `100` and I am not sure what that `100` refers to: is it number of tokens, number of batches, epochs? I inspected carefully the source code but since I am only superficially familiar with `C++`, I was not able to decipher the exact use of that parameter. I did, however, get the impression that it's being defined in a non-standard way. I was wondering if anybody can provide some insight about:

1. What exactly does that default value for `lrUpdateRate` refer to?
2. How can I deactivate `lrUpdateRate` during unsupervised training? Do I need to supply a really large number as its value or a really small one (e.g. `0`)?

I have used `fastText` extensively so far and this is a good opportunity to learn even more about its inner workings. Unfortunately, there is next to nothing on the web about this particular parameter.
